client/test/08-LEDcycling.c

```diff
@@ -0,0 +1,98 @@
+#include <stdio.h>
```

Comment:
how come this is file 08 while previous was 06? did you forget to commit a file?